docs/source/integrating/authentication.md

```diff
@@ -3,41 +3,39 @@ title: Authenticating with Talk
 permalink: /integrating/authentication/
 ---
 
-Out of the box Talk supports account registration with username and password, as well as features like forgot password. 
+### Authentication Strategies 
 
-You can also integrate Talk with any external authentication service that will enable
-seamless single sign-on (SSO) for users within your organization. There are a few
-methods of doing so:
+Talk comes with built in user management features like account registration with username/email and password, as well as features like resetting a forgotten password. 
 
-1. Passport Middleware
-2. Custom Token Integration
+There are four strategies to choose from when it comes to user authentication:
 
-Both methods work, but there are product decisions that will affect the overall
-choice.
+#### Talk Email Authentication
+Best when you want Talk to manage all user registration, creation and authentication. Requires the least amount of setup and configuration. Users register by creating a Username (must be unique across users, no spaces or special chars), Email, and Password and are stored in Talk’s database. Optionally - you can require an email verification step at time of account creation. *Plugins required: `talk-plugin-local-auth`, `talk-plugin-auth`*
```

Comment:
Instead of bolding the plugins, link them like this:

```suggestion
Best when you want Talk to manage all user registration, creation and authentication. Requires the least amount of setup and configuration. Users register by creating a Username (must be unique across users, no spaces or special chars), Email, and Password and are stored in Talk’s database. Optionally - you can require an email verification step at time of account creation. *Plugins required: [talk-plugin-auth](/talk/plugin/talk-plugin-local-auth), [talk-plugin-auth](/talk/plugin/talk-plugin-auth)*
```

Reply:
Good idea, I like that, will update the others too!